pkg/urbit/vere/io/ames.c

```diff
@@ -5,7 +5,8 @@
 #include "vere/vere.h"
 #include "ur/serial.h"
 
-
+#define FINE_FRAG 1024   //  bytes per fragment packet
+#define FINE_PAGE 16384  //  packets per page
```

Comment:
I like how this is just a runtime configuration thing, and the scry endpoint is agnostic, just "n through n+m".

Reply:
Worth considering adding `384` as a constant as well, since it comes up quite a bit below.

Reply:
Yeah, hopefully Vere's caching strategies can evolve without changing Ames's scry interface.

Good idea to add 384 as a constant.